src/snmpd/patch-5.7.3+dfsg/series

```diff
@@ -5,3 +5,4 @@
 0006-From-Jiri-Cervenka-snmpd-Fixed-agentx-crashing-and-or-freezing-on-timeout.patch
 0007-Linux-VRF-5.7.3-Support.patch
 0008-Enable-macro-DEB_BUILD_ARCH_OS-in-order-to-build-ipv.patch
+0009-fix-compilation-in-buster-env.patch
```

Comment:
In stead of this patch, could you try explore the new package [https://packages.debian.org/buster/snmpd](https://packages.debian.org/buster/snmpd)? We are wondering:
1. existing patches here are still applicable, or already included
2. basic runtime verification together with snmpagent.

I did a build of the package in sonic-slave-buster and it looks fine. Sorry for the long askes, this task is challenging.

Reply:
I checked the sources from https://packages.debian.org/source/buster/net-snmp and I see that the changes from 0007-Linux-VRF-5.7.3-Support.patch are not available.

Reply:
Any plan explore this direction or not?

Reply:
i agree with qi. I found the buster patch. snmptrapd_mysql_init from http://deb.debian.org/debian/pool/main/n/net-snmp/net-snmp_5.7.3+dfsg-5.debian.tar.xz

why not we use buster package since we are upgrade it to buster.

Reply:
I am investigating with another PR https://github.com/Azure/sonic-buildimage/pull/4590.
I will close that one after build and test.
You may have a look. I don't have the permission to directly push to your branch. Normally if PR owner select "Allow edits by maintainers" when submitting, I can push.

Reply:
The option "Allow edits by maintainers" is enabled.

I will also pull in the changes from #4590, build and test.

Reply:
I tested Mellanox binary and it works

Reply:
i updated to azure storage url.